manager/src/test/java/com/redhat/service/smartevents/manager/BridgesServiceTest.java

```diff
@@ -357,6 +362,28 @@ void testUpdateBridgeErrorHandlerWithNewErrorHandler() {
         assertThat(updatedResponse.getErrorHandler()).isNotNull();
     }
 
+    @Test
+    void testToResponse() {
+        Bridge bridge = createPersistBridge(READY);
+        bridge.setModifiedAt(ZonedDateTime.now(ZoneOffset.UTC));
+
+        BridgeResponse response = bridgesService.toResponse(bridge);
+
+        assertThat(response.getId()).isEqualTo(bridge.getId());
+        assertThat(response.getId()).isEqualTo(bridge.getId());
+        assertThat(response.getName()).isEqualTo(bridge.getName());
+        assertThat(response.getEndpoint()).isEqualTo(bridge.getEndpoint());
+        assertThat(response.getSubmittedAt()).isEqualTo(bridge.getSubmittedAt());
+        assertThat(response.getPublishedAt()).isEqualTo(bridge.getPublishedAt());
+        assertThat(response.getModifiedAt()).isEqualTo(bridge.getModifiedAt());
+        assertThat(response.getStatus()).isEqualTo(bridge.getStatus());
+        assertThat(response.getHref()).contains(bridge.getId());
```

Comment:
is this intentional?

Reply:
i do not see we set `href` as `id` in
```java
    protected Bridge createPersistBridge(String bridgeId, ManagedResourceStatus status) {
        Bridge b = Fixtures.createBridge();
        b.setId(bridgeId);
        b.setStatus(status);
        bridgeDAO.persist(b);
        return b;
    }
```

Reply:
Yes this is intentional. The `href` is set [here](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/f58125aec93e3041e44d504c25fb58b6e0e134e8/manager/src/main/java/com/redhat/service/smartevents/manager/BridgesServiceImpl.java#L293) and **contains** the `bridgeid`.